api/models/v0.3/modelAddress.js

```diff
@@ -98,18 +147,28 @@ class Address {
     return false;
   }
 
-  // TODO: Need to call validation API to validate the address
-  // which depends on Service Objects
-  validationresponse(isWorkAddress = undefined) {
+  /**
+   * validationResponse(isWorkAddress)
+   * TODO: Need to call validation API to validate the address
+   * which depends on Service Objects
+   *
+   * @param {boolean} isWorkAddress
+   */
```

Comment:
AAAH I didn't realize I was blocking you.  I'll take care of it next sprint.  Sorry about that.

Reply:
Well, once you create the validation APIs with Service Objects, you'd then implement this function, too.

Reply:
Cool, I'll stop feeling guilty now.